components/Voting/sessionPanel.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react'
+import { Badge, Buttons, Details, Icon, Panel, Title } from './sessionPanel.styled'
+
+import { Session } from '../../config/types'
+import SessionDetails from '../sessionDetails'
+
+interface SessionPanelProps {
+  session: Session
+  anonymousVoting: boolean
+  isVotedFor: boolean
+  isVotingDisabled: boolean
+  hideVotingButtons: boolean
+  isInShortlist: boolean
+  expandAll: boolean
+  toggleVote: (Session) => void
+  toggleShortlist: (Session) => void
+}
+
+export const SessionPanel: React.FC<SessionPanelProps> = ({
+  session: s,
+  anonymousVoting,
+  isVotedFor,
+  isVotingDisabled,
+  hideVotingButtons,
+  isInShortlist,
+  expandAll,
+  toggleVote,
+  toggleShortlist,
```

Comment:
Minor: extra colon

Reply:
The prettier config deliberately puts them in there

Reply:
Comma dangle is the best 😛